edx_proctoring/callbacks.py

```diff
@@ -15,10 +15,12 @@
 from rest_framework.response import Response
 from rest_framework.negotiation import BaseContentNegotiation
 
+from edx_proctoring.models import ProctoredExamStudentAttemptStatus as ExamAttemptStatus
```

Comment:
Nit: relocate this import to below edx_proctoring.api (alphabetical).  

Also any particular reason you are importing the model with an alternate name?  I usually only do this when a conflict exists, even if the class/method name is long.